src/autoload/CFG_GLPI.php

```diff
@@ -560,7 +560,7 @@
             'SolutionTemplate'       => ['tinymce'],
             'TaskTemplate'           => ['tinymce'],
             'ITILValidationTemplate' => ['tinymce'],
-            'Glpi\Form\Category'     => ['tinymce'],
+            \Glpi\Form\Category::class     => ['tinymce'],
```

Comment:
Why only this one is changed to the `::class` notation? All other should be as well.

Reply:
OK, changing those ones is a good idea now then ;p